artemis-hawtio/artemis-plugin/src/main/webapp/plugin/js/services/toolbar.js

```diff
@@ -25,7 +25,7 @@ var Artemis;
                     <form class="toolbar-pf-actions">
                         <div class="form-group toolbar-pf-filter">
                             <div class="input-group">
-                                <div class="input-group-btn">
+                                <div class="input-group-btn" style="padding-left: 10px">
```

Comment:
@andytaylor can you review this?

Reply:
sorry, the HTML code should not have been here, see comment below